### PR TITLE
swagger updates - task 6465636

### PR DESCRIPTION
**Pull request by srcharug**

### CODE REVIEW

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json [hunk 1]
```diff
@@ -7414,7 +7414,7 @@
           "$ref": "#/definitions/SubResource"
         }
       ],
-      "description": "Specifies information about the image to use. You can specify information about platform images, marketplace images, or virtual machine images. This element is required when you want to use a platform image, marketplace image, or virtual machine image, but is not used in other creation operations."
+      "description": "Specifies information about the image to use. You can specify information about platform images, marketplace images, or virtual machine images. This element is required when you want to use a platform image, marketplace image, or virtual machine image, but is not used in other creation operations. NOTE:Image reference publisher and offer can only be set when you create the scale set."
```
Comment (hyonholee) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-12-01/compute.json]: NIT: a space after NOTE:
Also, I think "when you create a scale set." is correct grammatically.

**Review by hyonholee — COMMENTED** [review]
Also, update previous versions of compute.json, if this description update also applies to previous versions.

**Comment by srcharug** [comment]
compute.json file is in most of the releases that go back till 2015. Should all the versions that has compute.json be updated?
Do you want me to include space after NOTE: in all the changes?

**Comment by hyonholee** [comment]
@srcharug  yes, could you update all previous compute.json file, if the same description applies to old versions?

**Comment by wangjun-jwa** [comment]
The purpose is to update MSDN doc with the new description. I think the MSDN doc is generated by latest compute.json file. @hyonholee, do we really need to update all versions for this?

**Comment by hyonholee** [comment]
@wangjun-jwa  It is better that old versions have the same description update if new description also applies to the old version.  Swagger is open source and anyone can create an SDK with old versions.

**Comment by srcharug** [comment]
@hyonholee I made the changes. Could you review the PR?

**Review by hyonholee — APPROVED** [review]

**Pull request merged**